src/styles/blocks/catalog.scss

```
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all $transition-time;
```

Do not set all propertie, set only those values you will change
```suggestion
    transition: transform $transition-time, color $transition-time;
```